pkg/apis/management.cattle.io/v3/authn_types.go

```diff
@@ -280,14 +280,15 @@ type GoogleOauthConfigApplyInput struct {
 type AzureADConfig struct {
 	AuthConfig `json:",inline" mapstructure:",squash"`
 
-	Endpoint          string `json:"endpoint,omitempty" norman:"default=https://login.microsoftonline.com/,required,notnullable"`
-	GraphEndpoint     string `json:"graphEndpoint,omitempty" norman:"required,notnullable"`
-	TokenEndpoint     string `json:"tokenEndpoint,omitempty" norman:"required,notnullable"`
-	AuthEndpoint      string `json:"authEndpoint,omitempty" norman:"required,notnullable"`
-	TenantID          string `json:"tenantId,omitempty" norman:"required,notnullable"`
-	ApplicationID     string `json:"applicationId,omitempty" norman:"required,notnullable"`
-	ApplicationSecret string `json:"applicationSecret,omitempty" norman:"required,type=password"`
-	RancherURL        string `json:"rancherUrl,omitempty" norman:"required,notnullable"`
+	Endpoint              string `json:"endpoint,omitempty" norman:"default=https://login.microsoftonline.com/,required,notnullable"`
+	GraphEndpoint         string `json:"graphEndpoint,omitempty" norman:"required,notnullable"`
+	TokenEndpoint         string `json:"tokenEndpoint,omitempty" norman:"required,notnullable"`
+	AuthEndpoint          string `json:"authEndpoint,omitempty" norman:"required,notnullable"`
+	TenantID              string `json:"tenantId,omitempty" norman:"required,notnullable"`
+	ApplicationID         string `json:"applicationId,omitempty" norman:"required,notnullable"`
+	ApplicationSecret     string `json:"applicationSecret,omitempty" norman:"required,type=password"`
+	RancherURL            string `json:"rancherUrl,omitempty" norman:"required,notnullable"`
+	FilterGroupMembership string `json:"filterGroupMembership,omitempty"`
```

Comment:
super minor and not blocking for me, but it looks like our naming convention for filters is of the form XFilter rather than FilterX.

Reply:
Ah. Will fix. The original PR had XFilter and it did not read nice to me. Did not know about any convention :(
Well, now I learned about it.

Reply:
Done with commit f329be6dcd32ad7a6623944341ee72986d771cbd